### PR TITLE
display detail movie, casts, trailer, overview

### PR DESCRIPTION
**Pull request by NVManh246**
https://edu-redmine.framgia.vn/issues/1355

### CODE REVIEW

**Comment by NVManh246** [comment]
ready

File: app/src/main/res/layout/item_video.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <ImageView
+            android:layout_width="140dp"
+            android:layout_height="100dp"
+            android:scaleType="centerCrop"
+            app:imageUrlYoutube="@{viewModel.observableVideo.key}"
+            />
+
+        <ImageView
+            android:layout_width="30dp"
+            android:layout_height="30dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_video.xml]: 23 24 hardcode

File: app/src/main/res/layout/item_video.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <android.support.constraint.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="5dp"
+        >
+
+        <ImageView
+            android:layout_width="140dp"
+            android:layout_height="100dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_video.xml]: 16 17 hardcode

File: app/src/main/res/layout/item_video.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <data>
+        <variable
+            name="viewModel"
+            type="com.framgia.nvmanh.boxmovie.screen.detail.ItemVideoViewModel"/>
+    </data>
+    <android.support.constraint.ConstraintLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:padding="5dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_video.xml]: hardcode

File: app/src/main/res/layout/activity_detail_movie.xml [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+                    app:layout_constraintLeft_toLeftOf="parent"
+                    app:layout_constraintRight_toRightOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/text_title_review"
+                    app:layoutManager="android.support.v7.widget.LinearLayoutManager"
+                    />
+
+                <Button
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:text="@string/title_viewmore"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_detail_movie.xml]: rename to title_view_more

**Pull request merged**